scripts/clan_resources/herb/herb_supply.py

```diff
@@ -0,0 +1,728 @@
+        else:
+            con_info = treated_cat.permanent_condition[condition]
+
+        amt_modifier = int(amount_used * .5) if int(amount_used * .5) >= 1 else 1
+
+        effect_message = ""
+        # apply mortality effect
+        if effect == HerbEffect.MORTALITY:
+            con_info[effect] += (
+                    3 * strength + amt_modifier
+            )
+            effect_message = i18n.t("screens.med_den.log_messages.mortality_down")
+
+        # apply duration effect
+        elif effect == HerbEffect.DURATION:
+            # duration doesn't get amt_modifier, as that would be far too strong an affect
+            con_info[effect] -= (
+                    1 * strength
+            )
+            if con_info["duration"] < 0:
+                con_info["duration"] = 0
+            effect_message = i18n.t("screens.med_den.log_messages.duration_down")
+
+        # apply risk effect
+        elif effect == HerbEffect.RISK:
+            for risk in con_info[effect]:
+                risk["chance"] += (
+                        3 * strength + amt_modifier
+                )
+                effect_message = i18n.t("screens.med_den.log_messages.risks_down")
+
+        if game.clan.game_mode == "classic":
+            # classic doesn't get logs
+            return
+
+        # create and append log message
+        message = i18n.t(
+            "screens.med_den.log_messages.herb_used",
+            herb=self.herb[herb_used].plural_display if amount_used > 1 else str('a ') + self.herb[
+                herb_used].singular_display,
+            condition=condition,
+            effect=effect_message
+        )
+
+        message = event_text_adjust(
+            Cat=treated_cat,
+            text=message,
+            main_cat=treated_cat,
+            clan=game.clan
+        )
+        self.log.append(message)
+
+    @staticmethod
+    def __apply_lack_of_herb(treatment_cat, condition: str, effect):
+        """
+        if the condition is a perm condition, give some consequence for not treated it
+        """
+        # TODO: this kinda feels like something that should happen within a theoretical condition class...
+
+        # only perm conditions and redcough can degenerate
+        if condition in treatment_cat.illnesses and condition != "redcough":
+            return
+        elif condition in treatment_cat.injuries:
+            return
+
+        # grab the correct condition dict so that we can modify it
+        con_info = treatment_cat.permanent_condition[condition]
+
+        if effect == HerbEffect.RISK:
+            for risk in con_info[effect]:
+                risk["chance"] -= randint(2, 4)
+                if risk["chance"] <= 1:
+                    risk["chance"] = 2
+        elif effect == HerbEffect.MORTALITY:
+            con_info[effect] -= randint(2, 4)
+            if con_info[effect] <= 1:
+                con_info[effect] = 2
+
+
+MESSAGES = load_lang_resource("screens/med_den_messages.json")
```

Comment:
```suggestion
```
I don't know what this is supposed to do and it scares me.

Reply:
Then you've gotta explain to me how I'm supposed to load resources, bc I'm banging pots and pans over here /lh